drop-transfer/src/protocol/v5.rs

```diff
@@ -44,6 +44,8 @@ pub struct File {
 pub struct TransferRequest {
     pub files: Vec<File>,
     pub id: uuid::Uuid,
+    #[serde(default)]
```

Comment:
What does `default` do in this case?

Reply:
Ensures backward compatibility. We cannot expect all the peers to support the change so we need to assume that when this field is not present then it's set to false.
Of course, the proper way of doing this would be to introduce another protocol version, but I was too lazy for this